about.html

```diff
@@ -0,0 +1,108 @@
+        </section>
+        <hr>
+        <section class="courses">
+            <div class="courses-header">
+                <h3>Course Categories</h3>
+                <hr>
+                <p>Our training courses are available in multiple formats, including virtual classroom.</p>
+            </div>
+            <div class="course-items">
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Building Information Modelling</h4>
+                        <p class="course-description">Building Information Modelling (BIM) and working with digital twins is becoming an increasingly important part of project design, improving the efficiency of works and easing building maintenance requirements after construction is complete.</p>
+                    </div>
+                        <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/925f25fe-93ff-4d15-a92c-797cc88b93a7/BIM.jpg?width=936" alt="Building Information Modelling (BIM)"> 
+                </article>
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Accredited Project Managemen</h4>
+                        <p class="course-description">These online training courses provide construction professionals with the key project management skills needed for the delivery of successful projects. As well as the two longer courses preparing for exams, we offer a number of bite-size eLearning modules for quick information on specific topics.</p>
+                    </div>
+                    <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/aa26ec9e-790a-4628-b1f6-e0a385d9fec6/collaborative-working.jpg?width=936" alt="Accredited Project Management">
+                </article>
+            </div>
+        </section>
+        <section class="partner">
+            <h3 class="partner-h">Partner</h3>
+            <hr>
+            <div class="partner-logo">
+                <ul>
+                    <li><img src="./image/logo-partner/allstate-logo.svg" alt="allstate-logo"></li>
+                    <li><img src="./image/logo-partner/asus-logo.svg" alt="asus-logo"></li>
+                    <li><img src="./image/logo-partner/atlassian-logo.svg" alt="atlassian-logo"></li>
+                    <li><img src="./image/logo-partner/autodesk-logo.svg" alt="autodesk-logo.svg"></li>
+                    <li><img src="./image/logo-partner/chase-logo.svg" alt="chase-logo"></li>
+                </ul>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <img src="./image/logo.png" alt="logo" class="logo-footer" width="150">
+        <p class="copy-right">2023 Hien Phan. <br>Some rights reserved</p>
+    </footer>
```

Comment:
- [ ] Kindly make sure you use the same background color, text color, spacing between elements,... on the footer of the **About** page like the one from the template design.

- [ ] Kindly make sure the design of the footer for both **About** and **Home** page sticks as much as possible to the one from the template design 

### Template design for about page
![Screenshot from 2023-01-05 20-23-14](https://user-images.githubusercontent.com/83042742/210853640-2fcf6e51-c1a8-4517-806b-f422df4bf2f0.png) 

### Template design for Home page 
![Screenshot from 2023-01-05 20-28-10](https://user-images.githubusercontent.com/83042742/210854100-4dc722b9-2d34-4a47-b9a3-bd7b269cbe51.png)

### Your Design for both pages 
![Screenshot from 2023-01-05 20-23-30](https://user-images.githubusercontent.com/83042742/210854192-eb50c123-86b1-4539-844c-a17c7ca41f61.png)